tools/vagrant/about.md

```diff
@@ -19,6 +19,15 @@ Vagrant works on top of some virtualization or containerization technology. To i
 - [Vagrant with VirtualBox](/tools/vagrant/vagrant-virtualbox.html)
 - [Vagrant with Docker](/tools/vagrant/vagrant-docker.html)
 
+## Usage
+
+  $ cd new_folder
+  $ vagrant init
```

Comment:
I generally give it some vagrant box I find, e.g. `vagrant init fedora/36-cloud-base`, then the Vagrantfile refers to that box without me having to specify it later on in the Vagrantfile.

Maybe let's recommend that?
```suggestion
  $ vagrant init fedora/36-cloud-base
```